lib/headers.sh

```diff
@@ -9,8 +9,8 @@ if [ -z "$TMUX_POWERLINE_DIR_HOME" ]; then
 	unset lib_dir
 fi
 
-source "${TMUX_POWERLINE_DIR_HOME}/config/helpers.sh"
 source "${TMUX_POWERLINE_DIR_HOME}/config/paths.sh"
+source "${TMUX_POWERLINE_DIR_HOME}/config/helpers.sh"
```

Comment:
Let's first ensure all the paths and corresponding env variables are available before doing other things. This way we can already source `${TMUX_POWERLINE_DIR_LIB}/util.sh` in `config/helpers.sh` and use `tp_err`.